src/tokenizer.ts

```diff
@@ -982,6 +998,26 @@ export class Tokenizer extends DiagnosticEmitter {
     }
   }
 
+  /**
+   * Skip any name token, whether or not it is a keyword.
+   */
+  skipIdentifierName(): bool {
```

Comment:
Could this become an `IdentifierHandling` option on `skip()`, maybe? :)

Reply:
Make sense, yeah this copied function is a bit ugly. I guess it felt a bit weird given the special case in `skip`.

How about this: I get rid of the special case in `skip` and specify it using an optional param like you suggested. Then, to make the common case the most natural to write, I'll make `skipIdentifier` as a shorthand for `skip(Token.IDENTIFIER, IdentifierHandling.PREFER)` and keep `skipIdentifierName` as a shorthand for `skip(Token.IDENTIFIER, IdentifierHandling.ALWAYS)`. Let me know what you think.